python/pyspark/sql/functions/builtin.py

```diff
@@ -3968,17 +4512,85 @@ def regr_syy(y: "ColumnOrName", x: "ColumnOrName") -> Column:
 
     Examples
     --------
-    >>> from pyspark.sql import functions as sf
-    >>> x = (sf.col("id") % 3).alias("x")
-    >>> y = (sf.randn(42) + x * 10).alias("y")
-    >>> spark.range(0, 1000, 1, 1).select(x, y).select(
-    ...     sf.regr_syy("y", "x")
-    ... ).show()
-    +-----------------+
-    |   regr_syy(y, x)|
-    +-----------------+
-    |68250.53503811...|
-    +-----------------+
+    Example 1: All paris are non-null
+
+    >>> import pyspark.sql.functions as sf
+    >>> from pyspark.sql.types import IntegerType, StructField, StructType
+    >>> schema = StructType([
+    ...   StructField('y', IntegerType(), True),
+    ...   StructField('x', IntegerType(), True)
+    ... ])
+    >>> df = spark.createDataFrame([(1, 2), (2, 2), (2, 3), (2, 4)], schema)
+    >>> df.select(sf.regr_syy("y", "x")).show()
+    +------------------+
+    |    regr_syy(y, x)|
+    +------------------+
+    |0.7499999999999999|
```

Comment:
```suggestion
    |0.7499999999999...|
```

Reply:
Can you adjust the input values to make the result more stable?
I suspect it may output `0.75` in some envs.